pages/comparison-tool.vue

```diff
@@ -0,0 +1,260 @@
+<template>
+  <div>
+    <ComparisonToolHero />
+    <div class="container">
+      <div class="comparison_card-container mx-auto my-5">
+        <div
+          v-for="(selectedFB, index) in selectedFormBuildersOption"
+          :key="selectedFB + index"
+        >
+          <ComparisonOptionSelectCard
+            :options="filteredOptions(index)"
+            :selected-option="selectedFB"
+            :card-number="index"
+            :form-builders-logo-src="formBuildersLogoSrc"
+            @change="handleOptionChange"
+          />
+        </div>
+      </div>
+      <FormBuilderComparisonDetails
+        :selected-form-builders-details="selectedFormBuildersDetails"
+        :selected-plans="selectedPlans"
+        :feature-name-list="featureNameList"
+        :form-builders="formBuilders"
+        @update:selectedPlans="updateSelectedPlans"
+      />
+      <!-- <div
+        v-if="!selectedFormBuildersOption.filter(Boolean).length"
+        class="my-5"
+      >
+        <h2 class="section__heading">Popular Form Builder Comparisons</h2>
+        <div class="section__grid">
+          <ComparisonCard
+            imgSrc="/form-building-platforms/formester-vs-typeform.svg"
+            formBuilder1="Formester"
+            formBuilder2="Typeform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+          <ComparisonCard
+            imgSrc="/form-building-platforms/jotform-vs-typeform.svg"
+            formBuilder1="Jotform"
+            formBuilder2="Typeform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+          <ComparisonCard
+            imgSrc="/form-building-platforms/formester-vs-jotform.svg"
+            formBuilder1="Formester"
+            formBuilder2="Jotform"
+            @onViewComparisonButtonClick="handleViewComparisonClick"
+          />
+        </div>
+      </div> -->
+    </div>
+  </div>
+</template>
+
+<script>
+import ComparisonToolHero from '@/components/comparision/ComparisonToolHero.vue'
+import ComparisonOptionSelectCard from '@/components/comparision/ComparisonOptionSelectCard.vue'
+import ComparisonCard from '@/components/comparision/ComparisonCard.vue'
+import FormBuilderComparisonDetails from '@/components/comparision/FormBuilderComparisonDetails.vue'
+import { featureNameList } from '@/constants/plan'
```

Comment:
Why is this a part of code? It beats the whole point of having a cms.

Reply:
yes, we would be moving the name to the cms.